NOTICE

```diff
@@ -10,3 +10,7 @@ Foundation (http://www.apache.org/).
 
 This product includes software developed by
 Joda.org (http://www.joda.org/).
+
+This product includes software developed by
```

Comment:
Is this a requirement from your company? We don't require contributors to add to this.

Reply:
Please see if your company requires this. If not, it would be best to remove this change.

In the meantime, I'm also checking with the OpenSearch project to determine if this is an acceptable change or not.